trk/src/Navigation/AppNavigator.js

```diff
@@ -98,6 +99,38 @@ function HomeStack() {
         component={DeveloperFeedbackForm}
         options={{ title: 'Developer Feedback', headerTitleAlign: 'center' }}
       />
+      <Stack.Screen
+        name="Feedback"
+        component={FeedbackForm}
+        options={{ title: 'Feedback Form', headerBackTitle: 'Climb Detail', headerTitleAlign: 'center' }}
+      />
+    </Stack.Navigator>
+  );
+}
```

Comment:
Github is behaving strangely. I only **removed** `Stack.Screen`s from HomeStack(), didn't add any.

Anyway, removed the navigations to `ClimbDetail` and `GlossaryDefinitions` as they are no longer accessible from `Home`